src/main/java/com/hartwig/platinum/scheduling/JobScheduler.java

```diff
@@ -15,7 +15,7 @@ static JobScheduler fromConfiguration(PlatinumConfiguration configuration, JobSu
                 .map(c -> c.delay().isPresent()
                         ? new TimedBatchScheduler(jobSubmitter, Delay.forMinutes(c.delay().get()), c.size())
                         : new ConstantJobCountScheduler(jobSubmitter, kubernetesClientProxy, c.size(), JobScheduler.DELAY_BETWEEN_SUBMISSIONS, JobScheduler.POLLING_INTERVAL))
-                .orElse(new ConstantJobCountScheduler(jobSubmitter, kubernetesClientProxy, 1, JobScheduler.DELAY_BETWEEN_SUBMISSIONS, JobScheduler.POLLING_INTERVAL));
+                .orElse(new ConstantJobCountScheduler(jobSubmitter, kubernetesClientProxy, 5, JobScheduler.DELAY_BETWEEN_SUBMISSIONS, JobScheduler.POLLING_INTERVAL));
```

Comment:
I think it should default to the number of pipelines to be submitted (or something like "unlimited"). Other than reruns most of the times no throttling is necessary.  I think the throttling should be an opt-in thing.